generators/app/tasks/appSetup/featuresFiles/firebasePerformanceSetup.js

```diff
@@ -1,8 +1,8 @@
 module.exports = function firebasePerformanceSetup() {
   let buildGradleContent = this.fs.read(`${this.projectName}/android/build.gradle`);
   buildGradleContent = buildGradleContent.replace(
-    'dependencies {',
-    "dependencies {\n\t\tclasspath 'com.google.firebase:perf-plugin:1.3.1'"
+    '// NOTE: Do not place your application dependencies here; they belong',
+    "classpath 'com.google.firebase:perf-plugin:1.3.1'\n\t\t// NOTE: Do not place your application dependencies here; they belong"
```

Comment:
the comment is duplicated and it seems incomplete, if im not wrong 😅

Reply:
The idea of this replace is to transform the string: `// NOTE: Do not place your application dependencies here; they belong` into `classpath 'com.google.firebase:perf-plugin:1.3.1'\n\t\t// NOTE: Do not place your application dependencies here; they belong` for adding the `firebase-performance` dependency
Before
```
...
// NOTE: Do not place your application dependencies here; they belong
...
```
After
```
...
classpath 'com.google.firebase:perf-plugin:1.3.1'
// NOTE: Do not place your application dependencies here; they belong
...
```